.github/actions/update-chart-version/action.yml

```diff
@@ -5,14 +5,23 @@ inputs:
     description: "The new version of the chart"
     required: true
   chart-dir:
-    description: "The path to the directory holding Chart.yaml"
+    description: "The relative path to the directory holding Chart.yaml"
     required: true
+  chart-repository-dir:
```

Comment:
same question here: why can't we just have one input with the path to the directory holding this file?

Reply:
Same as before.